src/commands/deploy.rs

```diff
@@ -687,26 +687,31 @@ async fn wait_for_ready(
     }
 }
 
-async fn is_ready(health_check_url: &str) -> bool {
-    let resp = reqwest::get(health_check_url).await;
-    let (msg, ready) = match resp {
-        Err(e) => (format!("error {}", e), false),
-        Ok(r) => {
-            let status = r.status();
-            let ok = status.is_success();
-            let desc = if ok { "ready" } else { "not ready" };
-            (format!("{desc}, code {status}"), ok)
+#[instrument(level = "debug")]
+async fn is_ready(app_info_url: &str, bindle_version: &str) -> Result<bool> {
+    // If the request fails, the app isn't ready
```

Comment:
...yet.  Looking at the call site I think this would cause the wait-for-ready loop to print a failure message and terminate, but couldn't this happen if, say, the domain is still being brought up?  Or there's a transient network error?  I'd prefer to err on the side of being forgiving here.

Reply:
Since we've just successfully used the network it seems much more likely for a failure here to be permanent(ish) than transient. _I_ would rather see an error immediately than a timeout in that situation, but we could probably come up with more nuanced error handling than either of those two options.

Reply:
I reverted to making failures non-fatal with additional WARN level logging